bylaws/010-funding.md

```diff
@@ -22,36 +30,87 @@ PHP-FIG SHALL NOT solicit donations from any party, especially from private indi
 
-### Through TideLift
+At any time, any member of the Core Committee MAY call an approval vote to remove a design or product whether actively sold or not, or to reinstate a previously removed design or product.
 
-Contributions SHALL be accepted through [TideLift](https://tidelift.com/). All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's TideLift account, which MUST be configured to pay out to the Open Collective account. 
+- If an approval vote passes, another approval vote SHALL NOT be called for the same design or product for a period of 30 days. 
+- If an approval vote passes causing a design or product to be reinstated, the Secretaries SHALL have the final say in whether the design or product will be sold.
+
+All three Secretaries, but only the Secretaries, MUST have full administrative access to PHP-FIG's print-on-demand providers.
 
 ## Spending money
 
-Funds donated to the PHP-FIG MUST be used solely for non-personnel, operating expenses.  PHP-FIG SHALL NOT pay individual contributors to PHP-FIG standards or other personnel, like Core Committee members, Secretaries, Project Representatives or working groups.
-All expenses MUST be approved by an Approval Vote of the Core Committee. The expense MUST include whether it is one time or recurring and at which frequency. All expenses MUST be justified by their contribution to the mission of PHP-FIG. In response to supplier price changes for previously accepted recurring expenses, Secretaries MAY request Implicit Approval from the Core Committee to raise the approved amount, only if that does not exceed a 10% increase.
+Funds donated to the PHP-FIG MUST be used solely for non-personnel operating expenses. The PHP-FIG MUST NOT pay individual contributors or other personnel, like Core Committee members, Secretaries, Project Representatives or working groups unless the payment is associated with an approved expense, such as reimbursing a Secretary that has paid for an approved expense out of pocket.
+All expenses MUST be approved by an Approval Vote of the Core Committee. The expense MUST include whether it is one time or recurring and the frequency at which the expense is incurred. All expenses MUST be justified by their contribution to the mission of PHP-FIG. In response to supplier price changes for previously accepted recurring expenses, Secretaries MAY request Implicit Approval from the Core Committee to raise the approved amount, only if that does not exceed a 10% increase.
 
 Expenses approved by the Core Committee MUST be listed below, and reported in the Open Collective budget.
 
-### Overfunding, and giving back to the community
+### Overfunding and disbursement
+
+Overfunding SHALL be defined as any amount raised that exceeds the amount needed to cover the existing PHP-FIG's yearly expenses, as defined in the "Approved expenses" section below, for a period of three years.
```

Comment:
```suggestion
Overfunding SHALL be defined as any amount raised that exceeds the amount needed to cover the existing PHP-FIG's yearly expenses, as defined in the "Approved expenses" section below, for a period of three years with a 10% cost increase year over year.  (For example, if the approved budget is $10, the maximum amount before funds are considered overfunding would be $10 + ($10 + $10 × 10% = $11) + ($11 + $11 × 10% = $11.10) = $32.10)
```

This would not affect my vote, but I think while we're updating this it would be a good idea to bake in forecasted inflation potentials into the war chest, and clarify what that calculation would be.

Reply:
The example calculation here is wrong, if you can update that I'd be happy to commit your suggestion.

Reply:
Good catch. I've updated it